cli/pack/systemd_dir.go

```diff
@@ -77,54 +76,71 @@ func initSystemdDir(packCtx *PackCtx, opts *config.CliOpts,
 }
 
 // systemdExecArgsForApp generates CLI arguments for start/stop commands in unit file.
-func systemdExecArgsForApp(inst running.InstanceCtx) (string, error) {
+func systemdExecArgsForApp(inst running.InstanceCtx) string {
 	if inst.SingleApp {
 		// There are no instances for single instance application. So only app name is returned.
-		return inst.AppName, nil
+		return inst.AppName
 	} else {
 		// Return <app><delimiter>%i format ars. %i will be replaced with instance name.
-		return fmt.Sprintf("%s%c%%i", inst.AppName, running.InstanceDelimiter), nil
+		return fmt.Sprintf("%s%c%%i", inst.AppName, running.InstanceDelimiter)
 	}
 }
 
+type systemdUnitParams struct {
+	AppName     string         `yaml:"AppName"`
+	ExecArgs    string         `yaml:"ExecArgs"`
+	TT          string         `yaml:"TT"`
+	ConfigPath  string         `yaml:"ConfigPath"`
+	FdLimit     uint64         `yaml:"FdLimit"`
+	InstanceEnv map[string]any `yaml:"instance-env"`
```

Comment:
It may work, but I don't sure.
```suggestion
	InstanceEnv map[string]string `yaml:"instance-env"`
```

Reply:
Done.